bundle/src/main/java/com/adobe/acs/commons/cmf/MigrationOptions.java

```diff
@@ -0,0 +1,27 @@
+package com.adobe.acs.commons.cmf;
+
+import javax.jcr.Session;
+
+public class MigrationOptions {
```

Comment:
Waiting to see how this is used; but the following seem like potential options:

* Commit batch size (if this is desirable; have to deal w multi-resource rollback then)
* Throttle (be it N ms every X resources or engage ACS's ThrottleTaskService)